sdk/monitor/perf.yml

```diff
@@ -19,6 +19,10 @@ parameters:
   displayName: Iterations (times to run each test)
   type: number
   default: '5'
+- name: Profile
+  displayName: Enable profiling
```

Comment:
```suggestion
```

In other languages we omit `displayName` because the parameter name is good enough.  Example:

https://github.com/Azure/azure-sdk-for-java/blob/77036c28abc8715b61fdad3f8e806028a72a3a21/sdk/storage/azure-storage-blob/perf.yml#L26-L28